lte/gateway/python/magma/mobilityd/ip_address_man.py

```diff
@@ -327,9 +330,18 @@ def alloc_ip_address(self, sid: str) -> Tuple[ip_address, int]:
 
             # Now try to allocate it from underlying allocator.
             ip_desc = self.ip_allocator.alloc_ip_address(sid, 0)
+            existing_sid = self.get_sid_for_ip(ip_desc.ip)
+            if existing_sid:
+                error_msg = "Got IP: {} for SID: {}, which already is " \
+                            "assigned to SID: {}".format(ip_desc.ip,
+                                                         sid,
+                                                         existing_sid)
+                raise DuplicateIPAssignmentError(error_msg)
```

Comment:
I don't see a corresponding catch in the RPC servicer, so this error case will crash mobilityd and cause it to restart. Is this the desired behavior?